pkg/agent/discoverycache.go

```diff
@@ -165,21 +165,21 @@ func shouldDeleteService(apiID, stage string) bool {
 
 func deleteConsumerInstanceOrService(consumerInstance *v1alpha1.ConsumerInstance, externalAPIID, externalAPIStage string) {
 	if shouldDeleteService(externalAPIID, externalAPIStage) {
-		log.Infof("API no longer exists on the dataplane; deleting the API Service and corresponding catalog item %s from Amplify Central", consumerInstance.Title)
+		log.Debugf("API no longer exists on the dataplane; deleting the API Service and corresponding catalog item %s from Amplify Central", consumerInstance.Title)
```

Comment:
I know these weren't listed in Cyrille's ticket about Info vs Debug, but a couple in here seem like they may need to be Info still

Reply:
I thought the same on a few of these delete-type messages.

Reply:
yeah, when we talked I had not thought of these. Maybe just the first ones in each path? Or the last ones?

Reply:
I think for consistency with the create messages, I will get rid of "Amplify Central" from the message.